pythonforandroid/bootstraps/sdl2/build/jni/Application.mk

```diff
@@ -5,3 +5,4 @@
 
 # APP_ABI := armeabi armeabi-v7a x86
 APP_ABI := $(ARCH)
+APP_PLATFORM := android-19
```

Comment:
I agree, this should not be added at all without an actual framework around what its value should be and how the user should control it. Setting it to a fixed value will work in some circumstances, but break others.

Reply:
This pr was labeled a WIP because it's a work in progress. It's this not a known protocol? Not ready to be merged because i am aware of these issues. When it's ready i will remove that WIP prefix :-)

Reply:
Yep I missed that one, cheers :+1: